syft/frameworks/torch/tensors/interpreters/derivatives.yaml

```diff
@@ -12,18 +12,25 @@
   self: grad
   other: grad if type(self) == type(other) else None
 
+- name: asin(self)
+  self: grad * (-self * self + 1).rsqrt()
+
 - name: mul(self, other)
   self: grad * other
   other: grad * self if type(self) == type(other) else None
 
-- name: sqrt(self)
-  self: grad / (2 * result)
-
-- name: asin(self)
-  self: grad * (-self * self + 1).rsqrt()
+- name: sigmoid(self)
+  self: self.sigmoid() * (1 - self.sigmoid())
```

Comment:
I'm not 100% of my maths but I suppose you should make `grad` happen as in other functions right ? Otherwise you give the derivative with respect to a self whose gradient is 1.

Reply:
I think autograd tensor does that automatically but I'm not sure

Reply:
I think autograd tensor does that automatically but I'm not sure

Reply:
Like I would expect `grad * self.sigmoid() * (1 - self.sigmoid())`
If this is added automatically however it's fine, but I'm not sure, considering other examples

Reply:
Now that you mention it, I agree

Reply:
my bad, thanks @LaRiffle!